packages/myst-transforms/src/enumerate.ts

```diff
@@ -229,29 +235,40 @@ export function formatHeadingEnumerator(counts: (number | null)[], prefix?: stri
   return out;
 }
 
-export interface IReferenceState {
-  file?: VFile;
-  initializeNumberedHeadingDepths: (tree: GenericParent) => void;
```

Comment:
This interface is now only for "resolver" - so these initialize/add methods are only on `ReferenceState`.